public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/view.php

```diff
@@ -103,6 +104,94 @@ function( $key, $question ) {
 	return $content;
 }
 
+/**
+ * Render a single feedback comment to a human-readable HTML string.
+ *
+ * @param WP_Comment|Feedback|string|int $comment A comment/feedback object or a comment ID.
+ * @param bool                           $echo    Whether to echo the output or return it. Default true.
+ *
+ * @return string The feedback in a question-answer HTML display (or empty string).
+ */
+function render_feedback_comment( $comment, $echo = true ) {
+	$feedback  = get_feedback_comment( $comment );
+	$questions = get_feedback_questions( $feedback->version );
+	$output    = '';
+
+	foreach ( $questions as $key => $question ) {
+		if ( 'rating' === $key ) {
+			continue;
+		}
+
+		$answer = $feedback->$key;
+
+		if ( $answer ) {
+			$output .= sprintf(
+				'<p class="speaker-feedback__question">%s</p><p class="speaker-feedback__answer">%s</p>',
+				wp_kses_data( $question ),
+				wp_kses_data( $answer )
+			);
+		}
+	}
+
+	if ( ! $echo ) {
+		return $output;
+	}
+
+	echo $output; // phpcs:ignore -- sanitized above.
+}
+
+/**
+ * Render the star rating for a given feedback comment.
+ *
+ * @param WP_Comment|Feedback|string|int $comment A comment/feedback object or a comment ID.
+ * @param bool                           $echo    Whether to echo the output or return it. Default true.
+ *
+ * @return string The feedback rating stars in HTML.
+ */
+function render_feedback_rating( $comment, $echo = true ) {
+	$feedback = get_feedback_comment( $comment );
+	$output   = render_rating_stars( $feedback->rating );
+
+	if ( ! $echo ) {
+		return $output;
+	}
+
+	echo $output; // phpcs:ignore -- sanitized in `render_rating_stars`.
+}
+
+/**
+ * Render a rating of X out of Y stars. Used by `render_feedback_rating`.
+ *
+ * @param int $rating    The selected rating value (stars below this will be "filled").
+ * @param int $max_stars The total rating value (will render this many stars).
+ *
+ * @return string The feedback rating stars in HTML.
+ */
+function render_rating_stars( $rating, $max_stars = 5 ) {
```

Comment:
If we're going to show an [average rating](https://github.com/WordPress/wordcamp.org/issues/347#issuecomment-610594528), we might need to adapt this to accept `$rating` as a float and be able to show a half star or something...

Reply:
I've been rounding to an int on the "average", no half-stars. Do you think we need half-stars?

Reply:
I dunno. It seems like a common practice. The Plugin Directory does it for plugin reviews. But... maybe not a big deal?

Reply:
I'll leave all this as-is for now, and not worry about it — that'd be easy enough to change later if we want to.